When I run `tox -e py35` on account-service I get the following error with the current ordering:

```
tests/test_api.py:3: in <module>
    import account_service.api
account_service/api.py:3: in <module>
    from falcon_mutualauth.mutualauth import Authorize
.tox/py35/lib/python3.5/site-packages/falcon_mutualauth/mutualauth.py:24: in <module>
    class MutualAuthRequest(object, Request):
E   TypeError: Cannot create a consistent method resolution
E   order (MRO) for bases Request, object
```

When I switch object and Request I don't get the error, @reaperhulk helped me figure this out earlier today

Switching the order doesn't affect the tests in py27 - so we figured it would not break anything in overcorps

--- a/falcon_mutualauth/mutualauth.py
+++ b/falcon_mutualauth/mutualauth.py
@@ -21,7 +21,7 @@ from twisted.web.server import Request
 logger = structlog.get_logger()
 
 
-class MutualAuthRequest(object, Request):
+class MutualAuthRequest(Request):
 
     """HTTP request received over mutually-authenticated TLS.
 
@@ -37,12 +37,6 @@ class MutualAuthRequest(object, Request):
     """
 
     roles_map = {}
-
-    def __init__(self, channel, queued):
-        # Separate the old-style class from new-style class to avoid unexpected
-        # MRO by explicitly calling the superclasses' __init__s separately
-        super(MutualAuthRequest, self).__init__()
-        Request.__init__(self, channel, queued)
 
     def _inject_headers(self):
         peer_cert = self.channel.transport.getPeerCertificate()

